src/actions/LoginActions.tsx

```diff
@@ -31,6 +31,9 @@ import { readLocalAccountSettings } from './LocalSettingsActions'
 import { registerNotificationsV2, updateNotificationSettings } from './NotificationActions'
 import { showScamWarningModal } from './ScamWarningActions'
 
+const PER_WALLET_TIMEOUT = 5000
+const MIN_CREATE_WALLET_TIMEOUT = 20000
```

Comment:
Isn't this a maximum timeout?

Reply:
not quite. its the minimum. we wait 5s per wallet being created but at minimum 20s. so we wait the following

1 wallet = 20s
2 = 20s
3 = 20s
4 = 20s
5 = 25s
6 = 30s

Reply:
Ah, then you should use `Math.max` on line 309.